docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,199 @@
+| Aanleiding     | Een consumer wil besluittypeinformatie van de besluiten die het resultaat kunnen zijn van het zaakgericht werken van de behandelende organisatie(s) opvragen.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om besluittypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus het besluittype is opgenomen. |
+| - gevolg       | Een overzicht met besluittype informatie zoals opgenomen in de catalogus wordt teruggegeven.             |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om specifieke besluittypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus het besluittype is opgenomen. Tevens moet de identficatie  van het besluittype wat uitgevraagd wordt bekend zijn. |
+| - gevolg       | Besluittypeinformatie zoals opgenomen in de catalogus wordt teruggegeven.             |
+| Gegevens       | BESLUITTYPE <br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/besluittype_read                                                                                |
+| Samenhang      | - |
+| Bijzonderheden | - |
+
+### Resource: INFORMATIEOBJECTTYPE
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil informatieobjecttypeinformatie van de informatieobjecten die voor kunnen komen bij het zaaktype zoals opgenomen in de catalogus van de organisatie opvragen.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om informatieobjecttypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus het besluittype is opgenomen. |
+| - gevolg       | Een overzicht van Informatieobjecttypeinformatie zoals opgenomen in de catalogus wordt teruggegeven.             |
```

Comment:
Het is toch een overzicht van informatieobjecttypen?

Reply:
Klopt ook, pas ik aan.

Reply:
Is aangepast